legend-engine-pure-code-compiled-core/src/main/resources/core/pure/graphFetch/graphFetch.pure

```diff
@@ -78,16 +78,27 @@ function meta::pure::graphFetch::execution::checkCompatibility(fe: FunctionExpre
             'Query returns: \'' + $fe.genericType.rawType.name->toOne() + '\'')
 }
 
-
-
 Class meta::pure::graphFetch::execution::AlloySerializationConfig
 {
+    {doc.doc = 'Give a custom typeKey name. Defaults to @Type'}
     typeKeyName : String[1];
+
+    {doc.doc = 'Set to true to include type info of the serialized class'}
     includeType: Boolean[0..1];
+
+    {doc.doc = 'Set to True to include path to element for Enums'}
     includeEnumType: Boolean[0..1];
+
+    {doc.doc = 'Give a valid string of dateTime format to serialize the dateTime properties respectively'}
+    dateTimeFormat: String[0..1];
+
     removePropertiesWithNullValues: Boolean[0..1];
     removePropertiesWithEmptySets: Boolean[0..1];
+
+    {doc.doc = 'Set to true to include fully qualified type info of the serialized class'}
     fullyQualifiedTypePath: Boolean[0..1];
+
+    {doc.doc = 'Set to True to include Alloy Relational Store Object Reference'}
```

Comment:
lets change this to
'Set to True to include Alloy Store Object Reference'

Reply:
Done